data/test_e2ebot/domain.yml

```diff
@@ -0,0 +1,16 @@
+version: "2.0"
+
+actions:
+ - utter_greet

```

Comment:
that's duplicate (stuff from `responses:` will automatically be added to `actions`)
```suggestion
```